lib/presentation/components/task_card.dart

```diff
@@ -9,36 +10,78 @@ class TaskCard extends StatefulWidget {
 }
 
 class _TaskCardState extends State<TaskCard> {
+  bool isSelected = false;
+
+  void toggleSelected() {
+    setState(() => isSelected = !isSelected);
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Row(
-      crossAxisAlignment: CrossAxisAlignment.end,
-      children: <Widget>[
-        TimelineStepper().padding(0, bottom: 10),
-        SizedBox(width: 20),
-        Expanded(child: TaskCardContent().padding(0, top: 20)),
-      ],
+    return GestureDetector(
+      onTap: toggleSelected,
+      child: TaskCardContent(
+              title: "Meditar",
+              schedule: "8:00 AM - 9:00 AM",
+              duration: "10 Minutes",
+              color: Colors.green,
+              isSelected: this.isSelected)
```

Comment:
Se vc colocar uma vírgula depois de `this.isSelected` o formatter vai jogar o parênteses pra baixo, acho que fica melhor

Reply:
Okay